lib/faraday/http_cache.rb

```diff
@@ -75,6 +95,7 @@ def initialize(app, options = {})
 
       @logger = options[:logger]
       @shared_cache = options.fetch(:shared_cache, true)
+      @instrumenter = options.fetch(:instrumenter, nil)
```

Comment:
There is no need to use `fetch` if the default is `nil`, you can simply grab the option like the `logger` one.